cosmos/x/evm/keeper/genesis.go

```diff
@@ -30,6 +30,10 @@ import (
 
 // InitGenesis is called during the InitGenesis.
 func (k *Keeper) InitGenesis(ctx sdk.Context, genState *core.Genesis) error {
+	if err := k.polaris.SetGenesisBlock(genState.ToBlock()); err != nil {
```

Comment:
if you are storing the genesisBlock by header in x/evm/plugins/block/genesis.go, why do you need to store it again here?